apps/financial-aid/web-veita/src/routes/Municipalities/municipalities.tsx

```diff
@@ -39,6 +49,28 @@ export const Municipalities = () => {
     }
   }, [data])
 
+  const [activityMunicipality] = useMutation(ActivityMunicipalityMutation)
+
+  const changeMunicipalityActivity = async (id: string, active: boolean) => {
+    await activityMunicipality({
+      variables: {
+        input: {
+          id,
+          active,
+        },
+      },
+    })
+      .then(() => {
+        getMunicipalities()
+        toast.success('Það tókst að uppfæra sveitarfélag')
```

Comment:
Are we sure about this pattern, getMunicipalities starts a skeleton loader and then we show a toast shouldn't we just show the loader here?